packages/teams-js/src/public/runtime.ts

```diff
@@ -370,6 +371,7 @@ export let runtime: Runtime | UninitializedRuntime = _uninitializedRuntime;
 export const versionAndPlatformAgnosticTeamsRuntimeConfig: Runtime = {
   apiVersion: 4,
   isNAAChannelRecommended: false,
+  isDeeplyNestedAuthSupported: false,
```

Comment:
Does this value need to be updated, if so, how it will be updated? For now, it looks it is just set to false.

Reply:
`versionAndPlatformAgnosticTeamsRuntimeConfig` is used as the default runtime for early versions of Teams that don't pass the runtime object. For others, all capabilities are dynamically discovered at runtime.